packages/viz/src/components/common/connectWithTransitionGroup.js

```diff
@@ -28,6 +55,9 @@ export default function connectWithTransitionGroup(connect) {
   ]
 
   willFunctions.forEach((key) => {
+    if(!connect || !connect.prototype){
```

Comment:
Why adding this condition ?

Reply:
I had a null pointer otherwise. But I think that this condition can be improved to be `!connect?.prototype`

Reply:
I think so 😉